loadtest/user/userentity/utils.go

```diff
@@ -73,6 +73,14 @@ func convertToTypedRoles(input []gqlRole) []*model.Role {
 	return roles
 }
 
+func convertToRoleString(input []gqlRole) string {
+	roles := ""
+	for _, r := range input {
+		roles += " " + r.Name
+	}
+	return roles
+}
```

Comment:
Another nit:
Why the space before and not after? I'd do something like
```suggestion
func convertToRoleString(input []gqlRole) string {
	roleNames := make([]string, 0, len(input))
	for _, r := range input {
		roleNames = append(roleNames, r.Name)
	}
	return strings.Join(roleNames, " ")
}
```

Reply:
Better :)